man/man8/zpool-wait.8

```diff
@@ -27,12 +27,13 @@
 .\" Copyright 2017 Nexenta Systems, Inc.
 .\" Copyright (c) 2017 Open-E, Inc. All Rights Reserved.
 .\"
-.Dd February 25, 2020
+.Dd May 27, 2021
 .Dt ZPOOL-WAIT 8
 .Os
+.
 .Sh NAME
 .Nm zpool-wait
-.Nd Wait for background activity to stop in a ZFS storage pool
+.Nd wait for activity to stop in a ZFS storage pool
```

Comment:
I think we should keep the word "background" here, as `zpool wait` does not wait for ordinary I/O to stop, AFAIK.

Reply:
Which is noted in the first and third sentence, then explained in a big table

Reply:
Fair enough. I mean, I still disagree, but I don't feel that strongly.